### ISSUE TITLE
DHT On/Off

### ISSUE THREAD

**Issue by nix87** [scakemyer/plugin.video.quasar#262]
Can we have an option to turn on/off DHT?

**Comment by scakemyer** [comment]
And why would you want that?

**Comment by nix87** [comment]
Because of private trackers. You need to have DHT turned off when you use them. ;)

**Comment by scakemyer** [comment]
So that's for #38, noted

**Comment by endlessterror** [comment]
@scakemyer since this request is already open, I'll avoid creating a new issue. I do not have any issues with private trackers. However, on my raspberry I had to remove pulsar because dht traffic started to become to much for my 1mbps ADSL connection, and despite that I did not keep torrents after playback.

I am not aware if quasar is any different in that area. Would it be feasible to have dht configured like rtorrent? That is to be on/off/auto, whereas the last option keeps dht enabled during searches and when torrent transfer takes place? If that is to complex, perhaps consider an on/off switch?
